MonoZelda/HUD/HUDManager.cs

```diff
@@ -0,0 +1,106 @@
+﻿using MonoZelda.Items.ItemClasses;
+using MonoZelda.Items;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Xna.Framework.Graphics;
+using Microsoft.Xna.Framework;
+using MonoZelda.Link;
+using System.Diagnostics;
+using MonoZelda.Collision;
+using MonoZelda.Sprites;
+using System.Reflection;
+using Microsoft.Xna.Framework.Content;
+using MonoZelda.Link.Projectiles;
+
+
+namespace MonoZelda.HUD
+{
+    public class HUDManager
+    {
+        private PlayerState _playerState;
+        private List<SpriteDict> hearts;
+        private ContentManager contentManager;
+        private ProjectileManager projectileManager;
+        private SpriteDict proj;
+
+        // Dictionary to map ProjectileType to sprite names
+        Dictionary<ProjectileType, string> projectileSpriteMap = new Dictionary<ProjectileType, string>
+        {
+            { ProjectileType.Arrow, "arrow" },
+            { ProjectileType.ArrowBlue, "arrow_blue" },
+            { ProjectileType.Boomerang, "boomerang" },
+            { ProjectileType.BoomerangBlue, "boomerang_blue" },
+            { ProjectileType.Bomb, "bomb" },
+            { ProjectileType.CandleBlue, "fire" }
+        };
```

Comment:
I feel like this could be a get method instead of a dictionary. Not sure what concrete benefits that has, but this dictionary is not changing at runtime so a switch statement inside of a method would do the same job.